schemas/skills/samples/complex-skillmanifest-2.1.preview-0.json

```diff
@@ -0,0 +1,184 @@
+    "$id": "SkillBot",
+    "name": "Sample skill definition that can handle multiple types of activities",
+    "version": "1.0",
+    "description": "This is a sample skill definition for multiple activity types",
+    "publisherName": "Microsoft",
+    "privacyUrl": "https://myskill.contoso.com/privacy.html",
+    "copyright": "Copyright (c) Microsoft Corporation. All rights reserved.",
+    "license": "",
+    "iconUrl": "https://myskill.contoso.com/icon.png",
+    "tags": [
+        "sample",
+        "travel",
+        "weather"
+    ],
+    "endpoints": [
+        {
+            "name": "americas",
+            "protocol": "BotFrameworkV3",
+            "description": "Production  endpoint for SkillBot in the Americas",
+            "endpointUrl": "http://myskill.contoso.com/api/messages",
+            "msAppId": "00000000-0000-0000-0000-000000000000"
+        },
+        {
+            "name": "eu",
+            "protocol": "BotFrameworkV3",
+            "description": "Production  endpoint for SkillBot in Europe",
+            "endpointUrl": "http://myskill.contoso.com/api/messages",
+            "msAppId": "11111111-0000-0000-0000-000000000000"
+        }
+    ],
+    "dispatchModels": {
+        "languages": {
+            "en": [
+                {
+                    "id": "SkillLuModel-en",
+                    "name": "SkillBot LU (English)",
+                    "contentType": "application/lu",
+                    "url": "http://sample.com/SkillBot-en.lu",
+                    "description": "English language model for the skill"
+                },
+                {
+                    "id": "SkillQnAModel-en",
+                    "name": "SkillBot QnA LU (English)",
+                    "contentType": "application/lu",
+                    "url": "http://sample.com/SkillBot-QnA-en.lu",
+                    "description": "English language model for the skill (QnAMaker"
+                }
+            ],
+            "es-ES": [
+                {
+                    "id": "SkillLuModel-es-ES",
+                    "name": "SkillBot LU (Spanish-Spain)",
+                    "contentType": "application/lu",
+                    "url": "http://sample.com/SkillBot-es-ES.lu",
+                    "description": "Spanish (Spain) language model for the skill"
+                }
+            ],
+            "es-MX": [
+                {
+                    "id": "SkillLuModel-es-MX",
+                    "name": "SkillBot LU (Spanish-Mexico)",
+                    "contentType": "application/lu",
+                    "url": "http://sample.com/SkillBot-es-MX.lu",
+                    "description": "Spanish (Mexico) language model for the skill"
+                }
+            ]
+        },
+        "intents": {
+            "bookFlight": "#/activities/bookFlight",
+            "getWeather": "#/activities/getWeather",
+            "*": "#/activities/message"
```

Comment:
Intents relate to what we had as actions before? So there are 2 - bookflight and getweather? * means anything else will just go to the root dialog for processing directly - as if the user typed a message directly to the bot?